tests/auxil/bot_method_checks.py

```diff
@@ -549,11 +549,9 @@ async def check_defaults_handling(
     }
     # We tested this for a long time, but Bot API 7.0 deprecated it in favor of
     # reply_parameters. In the transition phase, both exist in a mutually exclusive
-    # way. Testing both cases would require a lot of additional code, so we just
-    # ignore this parameter here until it is removed.
-    # Same for disable_web_page_preview
+    # way. Testing both cases would require a lot of additional code, so we for now are content
```

Comment:
which both cases?

Reply:
passing `allow_sending_without_reply` *and* testing `reply_parameters`